doc/nrf/libraries/others/emds.rst

```diff
@@ -9,45 +9,47 @@ Emergency data storage
 
 Overview
 ********
-The emergency data storage (EMDS) library provides persistent storage functionality designed to prevent the wear and tear of the flash memory.
+The emergency data storage (EMDS) library provides persistent storage functionality designed to prevent the wear and tear of the flash/RRAM.
```

Comment:
```suggestion
The emergency data storage (EMDS) library provides persistent storage functionality designed to prevent the wear and tear of the flash memory or RRAM.
```

Ideally change to this everywhere it appears, or something else perhaps, but I think "flash/RRAM", especially considering how often it appears in this file, breaks the flow of the text quite a bit, so I hope we can find something that works a bit better.

I have added several other comments for specific places where I have other suggested rewordings for those specific sentences.

Reply:
@peknis What are your thoughts on this? Any ideas for how to improve the flow a bit, or is it fine as it is?

Reply:
Otherwise your suggestion looks good, but you could have this intro before the Overview section. If there is nothing else noteworthy to tell about the library, you can leave out that section altogether. See https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/templates/library_template_README.html for further tips and hits on writing a library doc.